docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -49,9 +49,9 @@ Use the terminal or an Anaconda Prompt for the following steps:
 
       proceed ([y]/n)?
 
-  This creates the myenv environment in ``/envs/``. This
-  environment uses the same version of Python that you are
-  currently using because you did not specify a version.
+  This creates the myenv environment in ``/envs/``. No
+  packages will be installed in this environment. Python
```

Comment:
I would suggest removing `Python will be the Python of the base environment.` That's not really a good way to look about it. `conda create` doesn't _activate_ an environment, so all that's happening here is that you're staying in the same environment you are currently in. And strictly speaking that may not be the base environment. In fact, the current environment doesn't even need to have Python in it.

Reply:
Sentence removed. I kept it to stay in line with what was there. The text is better without.